File: js/programmers/level.0/prgm_181881_1.js
function solution(arr) {
  let result = 0;
  let copyArr = arr;
  while (true) {
    const changeArr = copyArr.map((a) => {
      if (a >= 50 && a % 2 === 0) return a / 2;
      if (a < 50 && a % 2 === 1) return a * 2 + 1;
      return a;
    });
    const checkArr = copyArr.every((a, i) => a === changeArr[i]);
    if (checkArr) break;
    result++;
    copyArr = changeArr;
  }
  return result;
}

File: python/programmers/level.0/prgm_181881_1.py
def solution(arr):
    answer = 0
    while(True):
        new = []
        for i in arr:
            if i >= 50 and i%2 == 0:
                new.append(int(i/2))
            elif i < 50 and i%2 == 1:
                new.append(i*2 + 1)
            else:
                new.append(i)

        if arr == new:
            return answer
        else:
            answer += 1
            arr = new